include/net/net_if.h

```diff
@@ -2101,6 +2109,27 @@ void net_if_unset_promisc(struct net_if *iface);
  */
 bool net_if_is_promisc(struct net_if *iface);
 
+/**
+ * @brief Check if there are any pending TX network data for a given network
+ *        interface.
+ *
+ * @param iface Pointer to network interface
+ *
+ * @return True if there are no any pending TX network packets for this network
```

Comment:
I just realized that the text here went wrong. The function returns true if there are pending packets, and false otherwise, now the description here says otherwise. @tbursztyka can you tweak this

Reply:
indeed, will do